### PR TITLE
Appended Archana's bio to Javascript track

### PR DESCRIPTION
**Pull request by archanid**

### CODE REVIEW

File: mentors/javascript/a.json [hunk 1]
```diff
@@ -54,5 +54,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I'm a sofware architect & engineer and I love to share knowledge, because I think that everybody should be able to learn how to develop software."
+  },
+  {
+    "github_username": "archanid",
+    "name": "Archana Sriram",
+    "link_text": "github",
```
Comment (SleeplessByte) [inline mentors/javascript/a.json]:
```suggestion
    "link_text": "GitHub",
```

> - Capitalization of GitHub is incorrect in "link_text" for archanid in mentors/javascript/a.json.

File: mentors/javascript/a.json [hunk 2]
```diff
@@ -54,5 +54,13 @@
...
+    "link_url": "https://github.com/archanid",
+    "avatar_url": "https://avatars0.githubusercontent.com/u/13165182?s=460&v=4",
```
Comment (SleeplessByte) [inline mentors/javascript/a.json]: If this is your regular GitHub avatar, you can use `null` so that it updates as you update your GitHub avatar.

Comment (archanid) [reply mentors/javascript/a.json]: Oh sweet!

**Comment by archanid** [comment]
@SleeplessByte changed all the `link_*` to null so it picks up the default gh stuff. Ready for another 👀

**Comment by SleeplessByte** [comment]
Welcome to JavaScript!

**Pull request merged**